src/main/resources/static/javascript/mainApp/admin.controller.js

```diff
@@ -26,17 +26,6 @@
 
```

Comment:
The ticket is to add Jasmine testing to the general.controller.js. If these changes are necessary (admin controller stuff), it should go into separate tickets.